lib/src/asset_loader.dart

```diff
@@ -3,6 +3,7 @@ import 'package:flutter/services.dart';
 abstract class AssetLoader {
   const AssetLoader();
   Future<String> load(String localePath);
```

Comment:
I think a more versatile signature for this method would be:
```
Future<Map<String, dynamic>> load(Locale locale);	
```
This way, the user can change any jsonParse logic and path resolution based on a locale, or even using plain maps.